### ISSUE TITLE
RSS feed link in sidebar should have email signup option as well as pretty button :-)

### ISSUE THREAD

**Issue by rufuspollock** [okfn/okfn.org#106]
Sign up by email with subscriber count is nice for RSS feed - any way to get it back (i know it disappeared a while ago ...)

**Comment by smth** [comment]
Yes, we can get it back, the issue is "do we want it?". I seem to recall it was a considered decision to remove email signup for the RSS feed. Has this now been reconsidered?

**Comment by rufuspollock** [comment]
Any link to discussion about desire to explicitly remove this. It did seem quite useful but if there was good reason to remove wouldn't want to reverse that ...

**Comment by smth** [comment]
If I remember correctly, the discussion I was involved in was a phone conversation between myself and @jwyg. The reason for removing was that it was confusing to have two email signups on the page, the mailing list was deemed more important, and people could subscribe to the RSS feed however they see fit, so long as they have the link.

As I say, this is from memory, so please correct me if I'm wrong @jwyg 

It would be good if all these things were documented here, maybe we should be more strict about where these discussions take place..
